holybro_qav250: add note about actuator configuration (#2388)
PX4/PX4-user_guide: en/frames_multicopter/holybro_qav250_pixhawk4_mini.md

  :::
  
- [Actuators](../config/actuators.md)

>   - Here is the main difference between the _discontinued_ Holybro Pixhawk 4 Mini QAV250 Kit and the new one based on the Pix32 v6.
>     - The default actuator configuration assumes that a flight controller with [I/O board](../hardware/reference_design.md#mainio-function-breakdown) such as the Pix32 v6 is used.
>     - The Pixhawk 4 Mini does not have the I/O board. Therefore, the actuator configuration MUST be updated: MAIN -> AUX.

Reviewer: Good catch. I am changing this though, because it is applied to the wrong line. The Geometry is correct for any frame, it is the wiring/output assignment that is no longer valid. I'll first revert and then add a second suggestion with the new comment.

```suggestion
  - Here is the main difference between the _discontinued_ Holybro Pixhawk 4 Mini QAV250 Kit and the new one based on the Pix32 v6.
 - You should not need to update the vehicle geometry.
```

  
- [Actuators](../config/actuators.md)
  - You should not need to update the vehicle geometry (as this is a preconfigured airframe).

>   - Assign actuator functions to outputs to match your wiring. 

Reviewer: @beniaminopozzan OK, so moved this to the line which makes sense. Are you sure this is correct? 

```suggestion
  - Assign actuator functions to outputs to match your wiring.
    - For the Pixhawk 4 Mini, and other controllers that do not have an [I/O board](../hardware/reference_design.md#mainio-function-breakdown), you will need to assign actuators to outputs on the `PWM AUX` tab.
    - The Pix32 v6 has an I/O board, so you can assign to either AUX or MAIN.    
```

Author: Thanks @hamishwillee !
